cli/src/main/java/org/jboss/as/cli/impl/AttributeNamePathCompleter.java

```diff
@@ -236,9 +236,11 @@ public int complete(String buffer, List<String> candidates, final ModelNode desc
             return -1;
         }
         final Collection<String> foundCandidates = handler.getCandidates(descr, writeOnly);
+
```

Comment:
Try to keep changes not related to your code to a minimum - such as new lines, deleted empty lines or adding some extra space. Those changes are shown and make reviewing your code harder and the file history less clear.

For example, the `ParserUtil.java` is flagged as changed, yet the only change is some extra space on 2 lines. Similar changes are in this class and in `OperationRequestCompleter.java`